From ec59aaea92214f320e068293d514caa4662af508 Mon Sep 17 00:00:00 2001
From: hololeap <hololeap@protonmail.com>
Date: Sat, 26 Oct 2024 10:15:59 -0600
Subject: [PATCH 1/1] Disable installation of doc archives

Remove the code that creates e.g.
/usr/share/doc/ghc-9.4.8/archives/libraries.html.tar.xz

This fixes the QA warning: "Notice: One or more compressed files were
found in docompress-ed directories."

Signed-off-by: hololeap <hololeap@protonmail.com>
--- a/src/Rules/Documentation.hs
+++ b/src/Rules/Documentation.hs
@@ -106,12 +106,6 @@ documentationRules = do
                    ++ [ html |    Haddocks   `Set.member` doctargets
                                || SphinxHTML `Set.member` doctargets ]
 
-                      -- include archives for whatever targets remain from
-                      -- the --docs arguments we got.
-                   ++ [ ar
-                      | (ar, doc) <- zip archives docPaths
-                      , archiveTarget doc `Set.member` doctargets ]
-
         need $ map (root -/-) targets
 
         when (SphinxHTML `Set.member` doctargets)
-- 
2.45.2

